pkg/units/ffi.go

```diff
@@ -136,16 +136,18 @@ func (e *FFI) SetMapElement(fd int, key uint32, value uint64) int {
 // ValidateProgram passes the program through the bpf verifier without executing
 // it. Returns feedback to the generator so it can adjust the generation
 // settings.
-func (e *FFI) ValidateEbpfProgram(prog []uint64) (*fpb.ValidationResult, error) {
-	if len(prog) == 0 {
+func (e *FFI) ValidateEbpfProgram(encodedProgram *fpb.EncodedProgram) (*fpb.ValidationResult, error) {
+	if len(encodedProgram.Program) == 0 {
```

Comment:
maybe also check that encodedProgram is not nil

Reply:
Done